### DISCUSSION TITLE
Webhooks definition repeated on Getting started page

### DISCUSSION THREAD

**Discussion by Exar04** [woodpecker-ci/woodpecker#2556] [Ask a Question]
At the start theres explaination on webhooks
<img width="800" alt="Screenshot 2023-10-09 at 11 59 01 PM" src="https://github.com/woodpecker-ci/woodpecker/assets/109629956/ee933e30-9ef9-484e-845d-d5ac8511dd86">

But then later after 4 lines thers again same definition
<img width="815" alt="Screenshot 2023-10-09 at 11 59 10 PM" src="https://github.com/woodpecker-ci/woodpecker/assets/109629956/35b494e0-b2bc-448b-8926-f30a002e4472">

Doesn't it seem redundant?

**Comment by anbraten** [comment]
@Exar04 True, that's pretty redundant. Could you cleanup the part in a PR?

**Reply by Exar04** [reply]
Yeah sure

**Reply by anbraten** [reply]
I leave that up to you.
